neva.sh

```diff
@@ -56,4 +56,4 @@ python3 \
 rm -fr ${TMPDIR}
 if test -f ${profile} ; then
   cp ${profile} ~/share/
-fi
+fi
```

Comment:
is your editor maybe dropping EOL whitespace? i'm not sure how to read github's ⛔ icon?
anyway not a big deal